tests/ibis/test_ibis_container.py

```diff
@@ -191,6 +210,60 @@ def test_column_values_are_unique(t_basic, t_schema_basic):
         modified_data.pipe(t_schema_basic.validate)
 
 
+@pytest.mark.parametrize(
```

Comment:
Adapted these tests from https://github.com/unionai-oss/pandera/blob/v0.25.0/tests/pandas/test_schemas.py#L1806-L1863 (doesn't exist for Polars backend).